lips/lip-7.md

```diff
@@ -0,0 +1,53 @@
+---
+lip: 7
+title: Technical Working Group Lifecycle
+authors: Viktor Bunin of Bison Trails (@viktorbunin)
+status: Draft
+type: Process
+created: 10/30/2020
+---
+
+
+## Abstract
+This LIP outlines the process by which technical working groups (TWG’s) are formed and how they should conduct their operations.
+
+## Formation of a Technical Working Group
+Technical Working Groups (TWG) are formed to address or explore specific technical topics or issues as they relate to Libra. TWGs collect information and expert input on a particular topic, discuss material changes to the Libra Payment Network (LPN), and create outputs in the form of recommendations, reports, and Libra Improvement Proposals (LIPs).
+
+The proposal to form a technical working group can be made by any Association member. The proposal must include, but is not limited to:
+The group’s scope and goal
+The limits of its focus
+The expected final output, typically as a LIP, report, or recommendation
+Expected duration of operation
+The initial facilitator (up to two individuals)
+A brief description of ideal desired member profiles so other Association members can join
+
+Once the proposal is created, the facilitator will circulate it in public slack channels relevant to the topic. The facilitator is also encouraged to proactively share the proposal with Association members that are likely to be impacted by this TWG or that could contribute to it. The facilitator must circulate the proposal over 1-2 weeks for sufficient interest to gather.
+
+After gathering interest, the facilitator must choose members to be part of the TWG, add them to the proposal, and identify their roles (co-facilitator, contributor, reviewer, etc.). The proposal must have representation from a minimum of 3 Association members to build collaboration and as a tool to determine relevance to the entire association, rather than an individual member.
```

Comment:
Why does the facilitator choose who will be part of the working group?  Is that to keep the groups small?  The reason I ask is that it could be odd if for example Bison Trails wants to join a working group but the facilitator says that they weren't chosen and thus can't participate in the group

Reply:
This is a great point. The intention here was to have the facilitator own drumming up support and interest, but not to be exclusionary. I will update to make it clear that membership is open to all. Thank you for pointing it out!